src/app-layer-protos.c

```diff
@@ -24,53 +24,18 @@
 
 #include "suricata-common.h"
 #include "app-layer-protos.h"
+#include "rust.h"
+
+AppProto AlprotoMax = ALPROTO_MAX_STATIC;
```

Comment:
style: should be snake case, probably `g_alproto_max` here. The `g_` to indicate it is a global.

Alternatively it could be replaced by a inline func `AlprotoMax()` which would return a lower scoped static variable.

Reply:
ok for g_alproto_max